bin/truffleruby

```diff
@@ -99,6 +122,9 @@ if [ $on_graalvm = false ]; then
 
 fi
 
+# no " to split $JAVA_OPTS into array elements
+java_args+=($JAVA_OPTS)
```

Comment:
Is this solely to avoid the duplication below?
Or does it serve another purpose like merging JAVA_OPTS and args classpath?